schema/src/main/scala/org/constellation/schema/snapshot/Snapshot.scala

```diff
@@ -5,11 +5,22 @@ import org.constellation.schema.signature.Signable
 
 import scala.collection.SortedMap
 
-case class Snapshot(lastSnapshot: String, checkpointBlocks: Seq[String], publicReputation: SortedMap[Id, Double])
-    extends Signable {
-  override def toEncode = checkpointBlocks :+ lastSnapshot
+case class Snapshot(
+  lastSnapshot: String,
+  checkpointBlocks: Seq[String],
+  publicReputation: SortedMap[Id, Double],
+  nextActiveNodes: NextActiveNodes,
```

Comment:
what if we will make a rollback and there are no such nodes in the cluster?

Reply:
Rollback case may not be handled yet. Essentially during rollback the NextActiveNodes should be ignored and we should start from the initial nodes - currently based on the whitelisting file. I mentioned it on the daily sometime ago.

Reply:
I guess it will be then `NextActiveNodes(Set.empty, Set.empty)` but @TheMMaciek  please confirm

Reply:
During rollback these will be set to some nodes, but we need to ignore them during rollback and use the initialFullNodes -> currently these are first 3 nodes in the whitelisting file. The initial nodes logic can be changed of course, especially if we will remove whitelisting.